instrumentation/sidekiq/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-sidekiq
 
+### Unrelease
+
+* FIXED: Ensure OpenTelemetry's Sidekiq `TracerMiddleware` is added to the front of the Client and Server middleware chains.
+
```

Comment:
We auto generate release notes from git commits as long as you follow Conventional Commits, so you do not have to manually set these. 

https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/main/CONTRIBUTING.md#use-conventional-commit-messages

Reply:
Ack. Updated the commit message and rebased this away.